skimage/io/_io.py

```diff
@@ -104,7 +104,7 @@ def imread_collection(load_pattern, conserve_memory=True,
                        plugin=plugin, **plugin_args)
 
 
-def imsave(fname, arr, plugin=None, **plugin_args):
+def imsave(fname, arr, check_contrast=False, plugin=None, **plugin_args):
```

Comment:
Until keyword only arguments are standard, this has to come after "plugin"

Reply:
Unless you want to suggest a change in default behavior, I suggest you set this to `True`.

Reply:
Yes, I propose a change in the default behaviour because of the before mentioned issue.